Temporary fix for the openSSL v3 incompatibility (#373)
owncloud/docs-server: modules/admin_manual/pages/installation/system_requirements.adoc

:ios-system-requirements-url: https://doc.owncloud.com/ios-app/ios_faq.html
:android-system-requirements-url: https://doc.owncloud.com/android/faq.html

> :description: Read this documentation for a recommended or supported environment for your  ownCloud Server. This documentation is not dependent if you plan a new installation or upgrade an existing one.

Reviewer: ```suggestion
:description: Read this documentation for a recommended or supported environment for your ownCloud Server. This documentation applies to both a new installation or upgrade of an existing installation.
```

I think this is what you intend to say?

== Introduction

{description}

== General Note

> There is an important note about a xref:installation/manual_installation/manual_installation_prerequisites.adoc#openssl-version[prerequisite of openSSL]. We highly recommend checking this prerequisite as it is not dependent on one Operating System.

Reviewer: ```suggestion
There is an important note about a xref:installation/manual_installation/manual_installation_prerequisites.adoc#openssl-version[prerequisite of openSSL]. We highly recommend checking this prerequisite as it needs to be checked regardless of the Operating System chosen.
```
